streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorTopologyTest.java

```diff
@@ -1569,24 +1064,6 @@ private Topology createSimpleTopologyWithDroppingPartitioner() {
                 .addSink("sink", OUTPUT_TOPIC_1, new DroppingPartitioner(), "processor");
     }
 
-    @Deprecated // testing old PAPI
-    private Topology createStatefulTopology(final String storeName) {
-        return topology
-            .addSource("source", STRING_DESERIALIZER, STRING_DESERIALIZER, INPUT_TOPIC_1)
-            .addProcessor("processor", define(new OldAPIStatefulProcessor(storeName)), "source")
```

Comment:
`define()` should be unused now and can be removed, too?

Reply:
Done.